go.mod

```diff
@@ -14,4 +14,4 @@ require (
 	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
 )
 
-go 1.12
+go 1.14
```

Comment:
Why not `1.15`?

Reply:
I think 1.14 is fine here as it's the minimal supported version.

https://golang.org/ref/mod#go-mod-file-go
`A go directive sets the expected language version for the module.`

Reply:
Makes sense. 👍